Pipfile

```diff
@@ -13,10 +13,15 @@ pywps = ">=4.2.6"
 nchelpers = "==5.5.7"
 netCDF4 = ">=1.5.4"
 beautifulsoup4 = "==4.9.3"
+rpy2 = "==3.3.6"
 
 [requires]
 python_version = "3.8"
 
 [dev-packages]
 pytest = "==5.4.3"
 black = "==19.10b0"
+Sphinx = "==2.4.4"
+setuptools = "*"
+wheel = "*"
+twine = "*"
```

Comment:
There may be a potential issue having these packages installed in this manner. There is no way to _just_ install the `dev` packages right now. Because of this we may want to leave the publishing package with `pip` as we don't need the packaging tools to be stored long term.